packages/lesswrong/server/reviewVoteUpdate.ts

```diff
@@ -300,3 +303,68 @@ export async function createVotingPostHtml () {
 //   AND "postedAt" < '2024-01-01'
 //   AND "positiveReviewVoteCount" > 0
 // ORDER BY "reviewVoteScoreHighKarma" DESC
+
+const createReviewWinners = async () => {
+  const posts = await Posts.find({
+    postedAt: {
+      $gte:moment(`${REVIEW_YEAR}-01-01`).toDate(), 
+      $lt:moment(`${REVIEW_YEAR+1}-01-01`).toDate()
+    }, 
+    finalReviewVoteScoreAllKarma: {$gte: 1},
+    reviewCount: {$gte: 1},
+    positiveReviewVoteCount: {$gte: 1}
+  }, {sort: {finalReviewVoteScoreHighKarma: 1}, limit: 1}).fetch()
+
+  const tagIdsFromPosts = posts.map(post => Object.keys(post.tagRelevance))
+  const tagIds = [...new Set(tagIdsFromPosts.flat())]
+  const tags = await Tags.find({_id: {$in: tagIds}}).fetch()
+
+  const adminContext = createAdminContext();
+
+  const coreTags: DbTag[] = await Tags.find({name: {$in: ["Rationality", "World Modeling", "World Optimization", "Practical", "AI"]}}).fetch() ?? []
```

Comment:
Missing `Community` and `Site Meta`, maybe deliberately?

Reply:
(also this should probably actually be `Tags.find({ core: true, deleted: false }).fetch()`, and then filter down by name, to avoid accidentally fetching stuff you didn't intend to fetch)